lib/resources/r4/communication.yaml

```diff
@@ -9,102 +9,86 @@ fields:
   description: The type of the FHIR resource.
   url: https://hl7.org/fhir/r4/Communication-definitions.html#Communication
   note: resourceType must be Communication.
+  action: create
   example: |
     {
       "resourceType": "Communication"
     }
-  action:
-    - create
 
-- name: inResponseTo
-  required: 'no'
-  cardinality: 0..1
-  type: Reference(Communication)
-  description: The Communication this Communication is in response to.
-  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.inResponseTo
-  note: 
-  example: |
-    {
-      "inResponseTo": {
-        "reference": "Communication/23456_45678",
-        "type": "Communicaiton"
-      }
-    }
-  action:
-    - create
 
 - name: status
   required: 'Yes'
   cardinality: 1..1
   type: code
-  description: The status of the communication
+  description: The status of the Communication.
   url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.status
   action: create
-  note: As Millenniums Message Center Communicaitons are instantaneously delivered, all communications must be in a "completed" status
+  note: All Communications must be in "completed" status.
```

Comment:
2 patterns going on. single and double quotes. Pick one

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/commits/720c35bf50e9cb8dd953ad8a2bce0b6a3c29f74a#diff-f08ae1e4c063b9a36a22adaa43fecf21L26-L65